### PR TITLE
Storage Supplier Produce QR Code

### PR DESCRIPTION
**Pull request by DanielmAaron**
- [x] Generate QR Code for storage handover (Available in view modal of storage handover)
- [x] Create Supplier Produce QR Code configuration dashboard
- [x] Update UI on QR Code Dashboard -Static

### CODE REVIEW

**Comment by DanielmAaron** [comment]
**Supplier Produce QR Code Dashboard**
Screenshot 2022-08-25 at 18 03 21
**View Modal on Supplier Produce QR Code Dashboard**
Screenshot 2022-08-25 at 18 03 27
**Static QR Code UI Updated**
Screenshot 2022-08-25 at 18 19 25

**Comment by DanielmAaron** [comment]
### UPDATED ICONS ON QRCODE DASHBOARDS
**STATIC QRCODE**
Screenshot 2022-08-26 at 11 56 54
**SUPPLIER PRODUCE QRCODE**
Screenshot 2022-08-26 at 11 57 02
**QRCODE ATTRIBUTE**
Screenshot 2022-08-26 at 11 56 43

**Comment by DanielmAaron** [comment]
### Updated Dashboard - rm storage viewmodal
Screenshot 2022-08-26 at 17 03 21

File: views/dashboard_qrcode_supplier.ejs [hunk 1]
```diff
@@ -37,8 +37,8 @@
           </div>
           <div class="card-body">
             <div class="card-text text-center">
-                  <b class="qrcode-card-title">Market Shortcode</b><div style="width: 15rem; height: 2rem; overflow: auto;"> <%=data[i].market_Shortcode%></div>
-                  <b class="qrcode-card-title">Market Quantity</b><div style="width: 15rem; height: 2rem; overflow: auto;"><%=data[i].market_quantity%>Kg</div>
+                  <b class="qrcode-card-title">FARM NAME</b><div style="width: 15rem; height: 2rem; overflow: auto;"> <%=data[i].market_Shortcode%></div>
```
Comment (jajukajulz) [inline views/dashboard_qrcode_supplier.ejs]: This needs to the farm name e.g. White Mountain Natural Produce (from WMNP)

**Comment by DanielmAaron** [comment]
### Farm Name Added to SupplierProduce QR Code Dashboard
Screenshot 2022-08-26 at 18 46 36

**Pull request merged**